lib/Logger.js

```diff
@@ -61,7 +70,7 @@ class Logger {
         const rLog = new RequestLogger(Config.logger,
                                        Config.level, Config.dump, Config.end,
                                        uids);
-        rLog.addDefaultFields({ name: this.name });
+        rLog.setParent(this);
```

Comment:
No, as it is now added to the default fields at either https://github.com/scality/werelogs/pull/86/files/3cd4272a971be31f77356bc3c0c3faa5bdfe67da#diff-87f1df4f7e51d77fc7d8943721be6631R40 or https://github.com/scality/werelogs/pull/86/files/3cd4272a971be31f77356bc3c0c3faa5bdfe67da#diff-87f1df4f7e51d77fc7d8943721be6631R44
The `name` field will thus be inherited through the DefaultFields properties.